src/views/AmalgamationSelection.vue

```diff
@@ -187,14 +187,9 @@ export default class AmalgamationSelection extends Vue {
 
   /** Called when component is created. */
   created (): void {
-    // if required data isn't set, go back to dashboard
+    // if required data isn't set, go to dashboard
     if (!this.getIdentifier) {
-      if (GetFeatureFlag('use-business-dashboard')) {
-        const dashboardUIUrl = `${this.getBusinessDashUrl}/${this.getIdentifier}`
-        navigate(dashboardUIUrl)
-        return
-      }
-      this.$router.push({ name: Routes.DASHBOARD })
+      this.navigateToBusinessDashboard(this.getIdentifier)
```

Comment:
Now this view is consistent with all the others (albeit, the other views are in Filings UI, not here).

Anyway, this is the correct logic.